packages/manager/modules/bm-server-components/src/network-tile/template.html

```diff
@@ -190,19 +190,19 @@
             >
                 <oui-action-menu-item
                     data-ng-if="!$ctrl.ola.isActivated()"
-                    data-ui-sref="app.dedicated-server.server.interfaces"
+                    data-ui-sref="{{ $ctrl.statePrefix }}.interfaces"
```

Comment:
```suggestion
                    data-ui-sref=".interfaces"
```

Reply:
It's applied for all places where i use child state.